src/Serilog.Sinks.AwsCloudWatch/CloudWatchLogSink.cs

```diff
@@ -67,10 +70,12 @@ public void Dispose()
             batchingSink.Dispose();
         }
 
+#if FEATURE_ASYNCDISPOSABLE
```

Comment:
According to this https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives we should be able to use something like 
```suggestion
#if NET8_0_OR_GREATER
```
And then we don't need the custom property, because this property is already defined by the framework.

Reply:
I've done that now, with NET6_0_OR_GREATER as that's the other target framework. I forgot that was an option as my company's coding style is to make specific feature flags as I did but I agree, this is better.